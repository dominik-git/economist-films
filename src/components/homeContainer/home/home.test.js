import React from 'react';
import renderer from 'react-test-renderer';
import Home from './home';

const data = [
  { title: 'xxx',
    id: '5',
    items:[
      {
        thumbnail:{
          url:'https://cdn.twivel.io/uploads/economist/episode/thumbnail/142/875X480.jpg'
        }
      }
    ],
   }
];

test('Home renders without data', () => {
  const tree : string = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Home renders with data', () => {
  const Home = jest.fn((props) => <div>hello</div>);
  const tree : string = renderer.create(<Home shelves={data}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
