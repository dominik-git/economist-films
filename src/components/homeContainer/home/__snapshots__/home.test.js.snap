exports[`test Home renders with data 1`] = `
<div>
  hello
</div>
`;

exports[`test Home renders without data 1`] = `
<div
  className="home">
  <button
    type="button">
    <a
      onClick={[Function]}
      style={Object {}}>
      Back
    </a>
  </button>
  <div />
  <p>
    Home page
  </p>
</div>
`;
